components/shared/profile/edit/EditProfile.tsx

```diff
@@ -0,0 +1,451 @@
+              Start Date{" "}
+            </label>
+            <Popover>
+              <PopoverTrigger asChild>
+                <Button className=" profile-shadow max-w-full grow bg-black-600 text-white-500">
+                  <Calendar size={16} /> Select Start Time
+                </Button>
+              </PopoverTrigger>
+
+              <PopoverContent className="z-20 w-full flex-none grow self-stretch rounded-[4px] bg-black-700 text-white-100">
+                <Controller
+                  control={control}
+                  name="availability.startTime"
+                  render={({ field: { onChange } }) => (
+                    <DayPicker
+                      key="from"
+                      className="bg-black-700 text-white-500"
+                      onSelect={(selectedDate) => onChange(selectedDate)}
+                      showOutsideDays
+                      weekStartsOn={0}
+                      selected={selectedFrom}
+                      defaultMonth={new Date()}
+                      mode="single"
+                      onDayClick={(selectedFrom) => {
+                        handleDaySelectFrom(selectedFrom);
+                      }}
+                    />
+                  )}
+                />
+              </PopoverContent>
+            </Popover>
+            <span className="paragraph-4-medium space-y-2 text-white-500">
+              Set to Local Time
+            </span>
+          </div>
+          <div className="flex w-full flex-col">
+            <label className="text-white-100" htmlFor="availability.endTime">
+              End Date
+            </label>
+            <Popover>
+              <PopoverTrigger asChild>
+                <Button className=" max-w-full grow bg-black-600 text-white-500">
+                  <Calendar size={16} /> Select End Time
+                </Button>
+              </PopoverTrigger>
+              <PopoverContent className="z-20  w-full flex-none  self-stretch  rounded-[4px] bg-black-700 text-white-100">
+                <Controller
+                  control={control}
+                  name="availability.endTime"
+                  render={({ field: { onChange, onBlur, value, ref } }) => (
+                    <DayPicker
+                      key="to"
+                      className="bg-black-700 text-white-500"
+                      onSelect={(selectedDate) => onChange(selectedDate)}
+                      showOutsideDays
+                      weekStartsOn={0}
+                      selected={selectedTo}
+                      defaultMonth={new Date()}
+                      mode="single"
+                      onDayClick={(selectedFrom) => {
+                        handleDaySelectTo(selectedFrom);
+                      }}
+                    />
+                  )}
+                />
+              </PopoverContent>
+            </Popover>
+            <span className="paragraph-4-medium space-y-2 text-white-300">
+              Set to Local Time
```

Comment:
Didn't we discuss how to fix these styles?


https://github.com/agordon123/git_note/assets/4108484/051ff8e9-e420-48bf-af23-f0d799bfb81f

Reply:
yep with the rdp tag, will be updated in my next push